docs/authentication/configuration/sign-in-options.mdx

```diff
@@ -1,5 +1,243 @@
+/>
+
+### Finished!
+
+Now, when a user signs up or signs in to your application, Clerk will send a verification link or code to the user's email.
+
+You can make changes to the redirect behavior of both Sign-in and Sign-up by visiting the **Paths** section of the Clerk Dashboard, or by passing in props if you are using Clerk's [custom components](/docs/components/overview).
+
+<Callout type="info">
+  For more information on one-time passcodes, see Clerk's [detailed guide](/docs/custom-flows/email-sms-otp).
+</Callout>
+
+</Steps>
+
+### Phone + SMS Code
+
+<Callout>
+  Phone number identifiers and SMS verification are Premium features that are only available on some plans. For more information visit the **Plan & Billing** section of the Clerk Dashboard.
+</Callout>
+
+Let's imagine that your application is going to be exclusively used by people on their phones. You want to collect the user's phone number as the identifier, and you want the user to sign-in to your app by entering a one-time code sent via SMS to the user's device.
+
+<Steps>
+
+### Allow users to sign-in using their phone number
+
+Ensure that **Phone number** is enabled as an identifier. Navigate to the **Email, Phone, Username** tab under the **User & Authentication** section in the Clerk Dashboard. The toggle next to **Phone number** should be on. If it's not, toggle it on and hit **Apply Changes**. Users can now sign-up and sign-in with their phone number.
+
+<Images 
+width={3024}
+height={1652}
+src="/docs/images/authentication/phone-number.png"
+alt="The 'Email, Phone, Username' tab with an arrow pointing to the toggle next to 'Phone number', toggled on."
+/>
+
+Note how the toggle for **Email address** is off, because in this example, your application is prioritizing a mobile-first approach.
+
+### Configure **Phone number** settings
+
+Select the gear symbol next to **Phone number** to open up the configuration settings. 
+
+<Images 
+width={3024}
+height={1652}
+src="/docs/images/authentication/phone-number-configuration.png"
+alt="The 'Email, Phone, Username' tab with an arrow pointing to the gear icon next to 'Phone number'."
+/>
+
+Because in this example, your application is mobile-first, you should have all of these options toggled on. Having all of these options toggled on will require users to provide a phone number on sign-up, require them to verify their phone number, and will allow them to verify their phone number with a code sent to their phone.
+
+<Images 
+width={3024}
+height={1652}
+src="/docs/images/authentication/phone-number-configuration-settings.png"
+alt="The 'Phone number' configuration settings with all options toggled on."
+/>
+
+### Allow users to sign-in with an SMS code
+
+In the **Authentication factors** section under **Email, Phone, Username**, toggle on **SMS verification code**. This will allows users to sign-in with a code sent to their phone.
```

Comment:
```suggestion
In the **Authentication factors** section under **Email, Phone, Username**, toggle on **SMS verification code**. This will allows users to sign in with a code sent to their phone.
```